src/components/shared/Breadcrumb/Breadcrumb.js

```diff
@@ -12,9 +13,17 @@ const BreadcrumbItem = ({ href, text }) => {
   return (
     <>
       {!!href ? (
-        <li>
-          <a href={href}>{text}</a>
-        </li>
+        href.startsWith('http') ? (
+          <li>
+            <a href={href}>{text}</a>
+          </li>
+        ) : (
+          <li>
+            <Link to={href} data-testid="internal-link">
+              {text}
+            </Link>
+          </li>
```

Comment:
What do you think about moving the `<li>` out of the condition?

```suggestion
        <li>
          href.startsWith('http') ? (
            <a href={href}>{text}</a>
          ) : (
            <Link to={href} data-testid="internal-link">
              {text}
            </Link>
          )          
        </li>
```

In a next step, we could replace it with a smarter component:

```suggestion
        <li>
          <SmartLink to={href}>
            {text}
          </SmartLink>
        </li>
```

Reply:
good idea!